src/content/blog/astro-320.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Astro 3.2: View Transitions improvements"
+description: "Astro 3.2 is out with several improvements to View Transitions, including the ability to control the history stack, and a JavaScript navigation API"
+publishDate: "September 28, 2023"
+authors:
+  - matthew
+  - martrapp
+coverImage: "/src/content/blog/_images/astro-320/blog-hero-3_2-release.jpg"
+socialImage: "/src/content/blog/_images/astro-290/og-image-2_9-release.jpg"
+lang: "en"
+---
+
+Astro 3.2 is out and several new improvments for [view transitions](https://docs.astro.build/en/guides/view-transitions/), 
+
+- **[History Replacement on Links](#history-replacement-on-links)**
+- **[JavaScript Navigation API](#javascript-navigation-api)**
+- **[Route announcer for screen readers](#route-announcer)**
+
+If you already have Astro installed, you can upgrade it to 3.2 by running the upgrade command in your project (using your package manager of choice):
+
+```
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## History Replacement on Links
+
+You can now configure your links to prevent a new history entry from being added to the browser stack. Using the `data-astro-history="replace"` attribute, Astro will instead use the underlying `history.replaceState` API. This means the user won't have to use the back button as many times for use-cases where lots of clicks are expected.
```

Comment:
The normal spelling would be "use case" (without hyphen)
```suggestion
You can now configure your links to prevent a new history entry from being added to the browser stack. Using the `data-astro-history="replace"` attribute, Astro will instead use the underlying `history.replaceState` API. This means the user won't have to use the back button as many times for use cases where lots of clicks are expected.
```